libs/state/src/lib/rx-state.ts

```diff
@@ -22,6 +28,23 @@ export type RxStateSetupFn<State extends object> = (
   >,
 ) => void;
 
+export type RxStateOptions = {
+  injector?: Injector;
+};
+
+function getInjectorFromOptions<
+  SetupFn extends Function,
+  Options extends { injector?: Injector },
+>(setupFnOrOptions?: SetupFn | Options, options?: Options) {
+  if (options) {
```

Comment:
Would it be safer to check `options?.injector` if we add new options in the future?